node/service/src/chain_spec.rs

```diff
@@ -183,18 +182,18 @@ pub fn testnet_genesis(
         }),
         democracy: Some(DemocracyConfig::default()),
         collective_Instance1: Some(CouncilConfig {
-            members: vec![],
+            members: initial_authorities.iter().map(|x| x.1.clone()).collect(),
             phantom: Default::default(),
         }),
         collective_Instance2: Some(TechnicalCommitteeConfig {
             members: vec![],
             phantom: Default::default(),
         }),
         elections: Some(ElectionsConfig {
-            members: vec![],
-            presentation_duration: 1 * DAYS,
-            term_duration: 28 * DAYS,
-            desired_seats: 0,
+            members: initial_authorities.iter().map(|x| (x.1.clone(), 1000000)).collect(),
+            desired_seats: 5,
+            presentation_duration: 25,
```

Comment:
These shouldn't be the final values for presentation/term duration.

Reply:
What would you change them to?